released/packages/coq-kruskal-theorems/coq-kruskal-theorems.1.0/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+synopsis: "Extending the Coq library for manipulating Almost Full relations with various forms of Kruskal's tree theorem"
+description: """
+   This library formalizes the high-level variants of Higman's theorem (for trees of bounded arity)
+   and Kruskal's theorem (for rose trees), depending on how these datatypes are implemented. Also,
+   Vazsonyi's conjecture to illustrate the expressive power of Kruskal's and Higman's theorem.
+"""  
+maintainer: ["Dominique Larchey-Wendling (https://github.com/DmxLarchey)"] 
+authors: "Dominique Larchey-Wendling (https://github.com/DmxLarchey)"
+license: "MPL-2.0"
+homepage: "https://github.com/DmxLarchey/Kruskal-Theorems/"
+bug-reports: "https://github.com/DmxLarchey/Kruskal-Theorems/issues"
+dev-repo: "git+https://github.com:DmxLarchey/Kruskal-Theorems/"
+
+build: [
+  [make "-j%{jobs}%" "type"]
+]
+install: [
+  [make "install"]
+]
+
+depends: [
+  "coq-kruskal-almostfull" {>= "1.0"}
+  "coq-kruskal-veldman" {>= "1.0"}
+]
+
+url {
+  src: "https://github.com/DmxLarchey/Kruskal-Theorems/archive/refs/tags/v1.0.tar.gz"
```

Comment:
Standard advice (but not required): since GitHub no longer guarantees stability of tarball, you can upload your own tarball into the GitHub release, as is done for Coq: https://github.com/coq/coq/releases/tag/V8.19.1

Reply:
Thx @palmskog for the advice. I did not know that tarballs were not frozen at release.

Reply:
I will take it into account in subsequent releases ...

Reply:
See the background here: https://github.blog/2023-02-21-update-on-the-future-stability-of-source-code-archives-and-hashes/

> GitHub will hold the source downloads byte-for-byte stable for no less than a year from today (February 21, 2023). This covers both tarball (.tar.gz) and zipball (.zip) formats.

Reply:
It seems that this is related to possible updates in the compression algorithm, or its options when compressing. Meaning that the uncompressed file would not be modified. Would there exist an option to associate the `sha256sum` to the uncompressed tarball or zipfile? On source code, it would not make much a computational difference to check for integrity there ...

Reply:
To my knowledge, opam doesn't currently support hashing of files inside the archive, the only packaging system I know that does this is Nix.